src/js/sample_analyses/frames/Frames.js

```diff
@@ -0,0 +1,117 @@
+if (typeof J$ === 'undefined') {
+    J$ = {};
+}
+
+(function (sandbox) {
+    sandbox.Frames = function () {
+        var Constants = sandbox.Constants;
+
+        var SPECIAL_PROP3 = Constants.SPECIAL_PROP3 + "M";
```

Comment:
Perhaps this should be a bit more specific to this analysis to avoid conflicts with other analyses?

Suggestion:
> var SPECIAL_PROP_FRAMES = Constants.SPECIAL_PROP3 + "FRAMES*";